src/modules/grubcfg/grubcfg.conf

```diff
@@ -11,3 +11,6 @@ defaults:
     GRUB_DISABLE_SUBMENU: true
     GRUB_TERMINAL_OUTPUT: "console"
     GRUB_DISABLE_RECOVERY: true
+
+# The plymouth theme to be set if plymouth binary is present
+plymouth_theme: manjaro-elegant
```

Comment:
Is there any other theme that comes with upstream plymouth that we can use as default?